src/queries/SystemIntakeQueries.ts

```diff
@@ -1,7 +1,6 @@
 import { gql } from '@apollo/client';
 
-// eslint-disable-next-line import/prefer-default-export
-export const CreateSystemIntake = gql`
+export default gql`
```

Comment:
This file is eventually going to have a few queries in it. Any chance you can leave it as a named export for the moment?